articles/architecture-scenarios/implementation/b2b/b2b-provisioning.md

```diff
@@ -20,35 +20,23 @@ useCase:
-:::
-
-When provisioning organizations you need to consider the following:
-
-* You will need to add the organization to your own application configuration and/or database
-* You will need to make changes to your Auth0 configuration.  This will include doing some or all of the following:
-    * Create a unique tenant
-    * Add a database connection (if you have isolated users per organization)
-    * Add an enterprise connection for this organization
-        * This will include working with the organization to either update their existing configuration or add configuration for your Auth0 tenant if they are not a legacy organization.
-    * Provision an administrator for the organization
+<%= include('../../_includes/_provisioning/_organizations.md', { platform: 'b2b' }) %>
 
 ## User migration
 
 <%= include('../../_includes/_provisioning/_user-migration.md', { platform: 'b2b' }) %>
 
 ## Provisioning organization users
 
-An organization should map directly to one of your business customers/partners.  Each business/partner that you are working with has users who will be logging in. We call those end users *organization users*.  
+An organization should map directly to one of your business customers/partners.  Each business/partner that you are working with has users who will be logging in. We call those end users *organization users*.
 There are two different approaches to how to store your organization users:
 
 * **Isolated to the organization**: Every user *belongs* to exactly one organization.  It would not make sense for that user to be a part of more than one organization, and even if they were, it would make sense for them to have a separate “identity” for that other organization. See [Provisioning Users Isolated to the Organization](#provisioning-users-isolated-to-the-organization) for more information.  For example, a retail employee that works part time at two different stores has two different logins for each of those stores even if the stores both use the SaaS application.
 * **Shared between organizations**: In a case like this, users either create credentials in your company, or they can access other organizations instances of your application using credentials from their own organization.  A simple way to look at this is that one user may be authorized to access more than one organization’s instance of the application.  A user would understand that they can use the same credentials to access both instances of an application. See [Provisioning Users Isolated to the Organization](#provisioning-users-isolated-to-the-organization) for more information. For example, some doctors contract with multiple clinics and may need to be able to sign into each separate clinic with their same credentials.
```

Comment:
The link is this paragraph is to the wrong header... It points to ** Provisioning users isolated to the organization** rather than ** Provisioning users shared between organizations**